modules/storages/app/components/storages/project_storages/members/row_component.rb

```diff
@@ -89,7 +86,12 @@ def principal_show_path
     end
 
     def storage_connection_status
-      return :not_connected unless oauth_client_connected?
+      unless oauth_client_connected?
```

Comment:
:yellow_circle: I think this method name is bad now. Therefore I suggest to remove the method and just use its body in the condition here. It is expressive enough and no need to think about a better method name.
```suggestion
      unless member.principal.remote_identities.exists?(integration: storage)
```

Reply:
I changed the method name to `storage_connected?`. I preferred to keep the method, because it keeps the abstraction level inside `storage_connection_status` more consistent.

After renaming, I noticed that I could replace the `unless` with an `if` by switching the method order around, so I did that as well.

Reply:
Okay, works for me. Thanks for taking care! :+1: